packages/website/ts/@next/components/link.tsx

```diff
@@ -13,19 +13,23 @@ interface LinkInterface {
     theme?: {
         textColor: string;
     };
+    target?: string;
```

Comment:
Can we change this to `shouldOpenInNewTab` because now this is being taken care of by our `Link` component in my branch. When I merge I'll just pass the boolean through..

Reply:
I'm cool w. changing this